### DISCUSSION TITLE
Resizing with both ALT and SHIFT pressed issues

### DISCUSSION THREAD

**Discussion by JustANarik** [aseprite/laf#25] [Q&A] [answered]
**FIRST ISSUE**
I use aseprite a lot, pretty much all my day, and sometimes this issue is just blowing my mind. for exmpl if I create a square, that is 32x32, then I select it and try to resize it holding both ALT and SHIFT (I'm on win 10), first thing it does, it reduces bottom and right side with one pixel, so it becomes 31x31, and that's so bad for me, and only then, it begins resizing from center, so for exmpl instead of resizing 32x32 sqr to 18x18 sqr, it resizes it to 17x17 or like that 

**SECOND ISSUE**
another issue is quite same, like when I resize a hollow rectangle or square with both ALT and SHIFT, sometimes some part of it just disappears !

**Comment by dacap** [comment, marked as answer]
Hi @NarayanTheProg, thanks for your post, actually I thought that you were using the *laf* library (this repository is for *laf*, not for *aseprite*). I've created one issue on Aseprite about your first problem:
* https://github.com/aseprite/aseprite/issues/2663

I'm not sure about the second one, but please refer to the [aseprite issues](https://github.com/aseprite/aseprite/issues) for these kind of problems.

**Reply by JustANarik** [reply]
Oh thatnks for such a fast answer :D

**Comment by JustANarik** [comment]
P.S sorry if I'll do anything wrong, I'm quite new to github and I'm only 13 years old :D
And I can have grammar mistakes because I'm not English :)

**Reply by dacap** [reply]
No problem, don't worry 👍

**Reply by JustANarik** [reply]
THanks :D

**Comment by JustANarik** [comment]
Hey, btw, u'r two brothers ? I mean creators of Aseprite: You and Gaspar

**Reply by dacap** [reply]
That's right! actually we are 3 brothers now ☺️
